### PR TITLE
Add smart contract feature example blog post

### PR DESCRIPTION
**Pull request by james-vendia**

### CODE REVIEW

**Review by brian-vendia — CHANGES_REQUESTED** [review]

File: posts/2022-05-25-smart-contract-feature-example.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+In this post, we'll look at one of the three smart contracts included in the feature example.  Specifically, and to highlight the ease of creating and invoking smart contracts using Vendia Share, we'll use the data validation smart contract example.   See the feature example itself for a complete step-by-step guide through all three smart contracts.
+
+
+## Example - Data Validation Smart Contract
+
+After the Lender and the Servicer create a Uni to share home loan information with each other, they want to maintain a high level of data quality.  Specifically, the Lender wants to ensure the Servicer is only able to view and act on validated loans.
```
[the remark this answers is not in the document]
Comment (james-vendia) [reply posts/2022-05-25-smart-contract-feature-example.md]: Addressed in latest

File: posts/2022-05-25-smart-contract-feature-example.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+The [feature example](https://github.com/vendia/examples/tree/main/features/share/smart-contracts) demonstrates programmatic execution using a set of provided scripts.  For the rest of this post, we'll use the Vendia Share web application to make the invocation and results more visual.
+
+From the **LenderNode** of the [Uni created](https://github.com/vendia/examples/tree/main/features/share/smart-contracts#step-1---create-a-universal-application) in the feature example, click the `Smart Contracts` widget to get a listing of all available smart contracts.
+
+<figure>
+    <img src="https://user-images.githubusercontent.com/85032783/170310807-9cf127d4-cfa0-4ed5-890b-a8500ac5f071.png" />
+    <figcaption ><b>Figure 1</b> - <i>Smart Contract Listing</i></figcaption>
+</figure>
+
+Select the `validation-smart-contract` to view its details, including the ability to manually invoke the smart contract.  Next, click the `Invoke` button in the upper right and provide a valid `loanIdentifier`. The `loanIdentifier` provided is the value that will be sent to the parameterized `inputQuery` defined in the previous section.
```
Comment (db-vendia) [inline posts/2022-05-25-smart-contract-feature-example.md:197]: Nitpick, but you you say "click the `Invoke` button in the upper right" but in the modal (which is not the screen to which you are referring to, but is the screen highlighted in the screenshot) the Invoke button is in the lower right. Is it worth adding another screenshot to show the Invoke button in the upper right then this screenshot showing the input into the SC?

Comment (db-vendia) [reply posts/2022-05-25-smart-contract-feature-example.md:197]: The rest looks good.

File: posts/2022-05-25-smart-contract-feature-example.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+Selecting one of the ledger entries provides more detail about the ledgered transaction.  For example, the output mutation generated by the validation smart contract can be seen in its entirety.
+
+<figure>
+    <img src="https://user-images.githubusercontent.com/85032783/170310802-af3ae615-2dc4-476a-bdf2-46b8caecdeea.png" />
+    <figcaption ><b>Figure 6</b> - <i>Smart Contract Output Mutation Details</i></figcaption>
+</figure>
+
+## Conclusion
+
+Smart contracts can be used for a variety of purposes, including data validation, computation, and enrichment.  The recently published [smart contract feature example](https://github.com/vendia/examples/blob/main/features/share/smart-contracts/README.md) provides a step-by-step guide to create and invoke a set of smart contracts, each demonstrating slightly different interaction patterns and smart contract composition approaches.  Using smart contracts allows data _and code _to be shared and ledgered for real-time awareness and full transparency across a network of Uni participants.
```
Comment (james-vendia) [inline posts/2022-05-25-smart-contract-feature-example.md:239]:
```suggestion
Smart contracts can be used for a variety of purposes, including data validation, computation, and enrichment.  The recently published [smart contract feature example](https://github.com/vendia/examples/blob/main/features/share/smart-contracts/README.md) provides a step-by-step guide to create and invoke a set of smart contracts, each demonstrating slightly different interaction patterns and smart contract composition approaches.  Using smart contracts allows data _and code_ to be shared and ledgered for real-time awareness and full transparency across a network of Uni participants.
```

**Pull request merged**